DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateApprovals/DelegateRejectionPage.cshtml

```diff
@@ -40,6 +40,12 @@
           @Model.DateRegistered?.ToShortDateString()
         </dd>
       </div>
+      <div class="nhsuk-summary-list__row basic-summary-list__row">
+        <dt class="nhsuk-summary-list__key">
+          Professional Registration Number
+        </dt>
+        <partial name="_SummaryFieldValue" model="@Model.ProfessionalRegistrationNumber" />

```

Comment:
Just noticed that we get a "Possible null assigned to non-nullable entity" warning on these partial models, when we know full well that the model could be null because that's what the partial is about. Is there a way to set this up so we don't get the warning here? Looks like the partial's view model is a nullable string, so I'm not sure there is much we can do here.

Reply:
I can't see any way to avoid this warning other than manually disabling them with resharper each time/each file. We don't want to disable it solution wide though. It doesn't do this in Rider, so I tried updating visual studio and resharper, but it made no difference.

Reply:
Fair enough, that's the answer I was expecting. I don't think there are many of us still using visual studio, so probably not worth bothering spending any extra time on this.